tests/linkediting.js

```diff
@@ -625,9 +824,20 @@ describe( 'LinkEditing', () => {
 		} );
 
 		describe( 'upcast converter', () => {
+			let element, editor;
+
+			beforeEach( () => {
```

Comment:
Another nested `beforeEach` might be probably safe as it handles different editors. I don't like that there are two editors on the same page in this tests.

![](https://user-images.githubusercontent.com/247363/79731441-91522e00-82f2-11ea-97b3-5a37c54e5707.png)

Again, this does not break now, but we need at least a follow-up for this whole file.